solutions/swb-reference/SWBv2.postman_collection.json

```diff
@@ -817,7 +817,19 @@
             "url": {
               "raw": "{{API_URL}}/projects",
               "host": ["{{API_URL}}"],
-              "path": ["projects"]
+              "path": ["projects"],
+              "query": [
+                {
+                  "key": "paginationToken",
+                  "value": "eyJwayI6IkVUQyIsInNrIjoiRVQjNmM3MzJlMTEtODdmYi00MGU3LWFlM2ItMjU1MTY1OGQ3OGYwRVRDIzQwYjAxNTI5LTBjN2YtNDYwOS1hMWUyLTcxNTA2OGRhNWYwZSJ9",
```

Comment:
Do we need to provide this value in postman collection?

Reply:
It's just an example. Would you like me to provide a more obvious value as an example?

Reply:
If you were to use this value would it work? I suggest using for example values only what will work, e.g. pageSize makes total sense, while for token - probably leave it blank